<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="7" passed="4" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-10-04T23:52:14 IST" name=" Master suite" finished-at="2025-10-04T23:54:05 IST" duration-ms="111857">
    <groups>
      <group name="regression">
        <method signature="Tc_001AccountRegistrationtest.Testregistration()[pri:0, instance:testcases.Tc_001AccountRegistrationtest@2d7275fc]" name="Testregistration" class="testcases.Tc_001AccountRegistrationtest"/>
        <method signature="Tc_002Accountlogin.testlogin()[pri:0, instance:testcases.Tc_002Accountlogin@36bc55de]" name="testlogin" class="testcases.Tc_002Accountlogin"/>
      </group> <!-- regression -->
      <group name="sanity">
        <method signature="Tc_002Accountlogin.testlogin()[pri:0, instance:testcases.Tc_002Accountlogin@36bc55de]" name="testlogin" class="testcases.Tc_002Accountlogin"/>
      </group> <!-- sanity -->
      <group name="Datadriven">
        <method signature="Tc_003Logindatadriventestcase.login(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testcases.Tc_003Logindatadriventestcase@564fabc8]" name="login" class="testcases.Tc_003Logindatadriventestcase"/>
      </group> <!-- Datadriven -->
      <group name="master">
        <method signature="Tc_001AccountRegistrationtest.Testregistration()[pri:0, instance:testcases.Tc_001AccountRegistrationtest@2d7275fc]" name="Testregistration" class="testcases.Tc_001AccountRegistrationtest"/>
        <method signature="Tc_002Accountlogin.testlogin()[pri:0, instance:testcases.Tc_002Accountlogin@36bc55de]" name="testlogin" class="testcases.Tc_002Accountlogin"/>
      </group> <!-- master -->
    </groups>
    <test started-at="2025-10-04T23:52:14 IST" name="test" finished-at="2025-10-04T23:54:05 IST" duration-ms="111857">
      <class name="testcases.Tc_002Accountlogin">
        <test-method is-config="true" signature="setup(java.lang.String,java.lang.String)[pri:0, instance:testcases.Tc_002Accountlogin@36bc55de]" started-at="2025-10-04T23:52:20 IST" name="setup" finished-at="2025-10-04T23:52:22 IST" duration-ms="2099" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="testlogin()[pri:0, instance:testcases.Tc_002Accountlogin@36bc55de]" started-at="2025-10-04T23:52:22 IST" name="testlogin" finished-at="2025-10-04T23:52:24 IST" duration-ms="1684" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testlogin -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:testcases.Tc_002Accountlogin@36bc55de]" started-at="2025-10-04T23:52:24 IST" name="teardown" finished-at="2025-10-04T23:52:24 IST" duration-ms="215" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- testcases.Tc_002Accountlogin -->
      <class name="testcases.Tc_001AccountRegistrationtest">
        <test-method is-config="true" signature="setup(java.lang.String,java.lang.String)[pri:0, instance:testcases.Tc_001AccountRegistrationtest@2d7275fc]" started-at="2025-10-04T23:52:14 IST" name="setup" finished-at="2025-10-04T23:52:17 IST" duration-ms="3356" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Testregistration()[pri:0, instance:testcases.Tc_001AccountRegistrationtest@2d7275fc]" started-at="2025-10-04T23:52:17 IST" name="Testregistration" finished-at="2025-10-04T23:52:20 IST" duration-ms="2574" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Testregistration -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:testcases.Tc_001AccountRegistrationtest@2d7275fc]" started-at="2025-10-04T23:52:20 IST" name="teardown" finished-at="2025-10-04T23:52:20 IST" duration-ms="288" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- testcases.Tc_001AccountRegistrationtest -->
      <class name="testcases.Tc_003Logindatadriventestcase">
        <test-method is-config="true" signature="setup(java.lang.String,java.lang.String)[pri:0, instance:testcases.Tc_003Logindatadriventestcase@564fabc8]" started-at="2025-10-04T23:52:24 IST" name="setup" finished-at="2025-10-04T23:52:27 IST" duration-ms="2379" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Windows]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="login(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testcases.Tc_003Logindatadriventestcase@564fabc8]" started-at="2025-10-04T23:52:28 IST" name="login" data-provider="LoginData" finished-at="2025-10-04T23:52:30 IST" duration-ms="2284" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[fun123@gmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Test@123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Valid]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="login(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testcases.Tc_003Logindatadriventestcase@564fabc8]" started-at="2025-10-04T23:52:30 IST" name="login" data-provider="LoginData" finished-at="2025-10-04T23:53:01 IST" duration-ms="30672" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[manideep@909]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Test@123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[invalid]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at testcases.Tc_003Logindatadriventestcase.login(Tc_003Logindatadriventestcase.java:51)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="login(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testcases.Tc_003Logindatadriventestcase@564fabc8]" started-at="2025-10-04T23:53:01 IST" name="login" data-provider="LoginData" finished-at="2025-10-04T23:53:32 IST" duration-ms="31243" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[fun22@yopmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Test@123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[invalid]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at testcases.Tc_003Logindatadriventestcase.login(Tc_003Logindatadriventestcase.java:51)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="login(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testcases.Tc_003Logindatadriventestcase@564fabc8]" started-at="2025-10-04T23:53:32 IST" name="login" data-provider="LoginData" finished-at="2025-10-04T23:54:03 IST" duration-ms="30623" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[fun33@yopmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Test@123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[invalid]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at testcases.Tc_003Logindatadriventestcase.login(Tc_003Logindatadriventestcase.java:51)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:308)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:71)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:113)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="login(java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testcases.Tc_003Logindatadriventestcase@564fabc8]" started-at="2025-10-04T23:54:03 IST" name="login" data-provider="LoginData" finished-at="2025-10-04T23:54:05 IST" duration-ms="2088" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Fun143@yopmail.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Test@123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[valid]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:testcases.Tc_003Logindatadriventestcase@564fabc8]" started-at="2025-10-04T23:54:05 IST" name="teardown" finished-at="2025-10-04T23:54:05 IST" duration-ms="256" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- testcases.Tc_003Logindatadriventestcase -->
    </test> <!-- test -->
  </suite> <!--  Master suite -->
</testng-results>
